src/Logger_int.java

```diff
@@ -0,0 +1,340 @@
+package com.aap.gitst;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This file "Logger_int.java" should be located in "com\aap\gitst" directory, not in "C:\Users\bchauhan\Desktop\NBIOT\testing\Testing\src". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1598)